docs/client-development/authentication.rst

```diff
@@ -78,3 +78,169 @@ This would then be used in an API call like:
     the application.
 
 .. _Standard: https://vng-realisatie.github.io/gemma-zaken/
+
+Generating JWT Examples:
+------------------------
+
+Example of how to generate JWT tokens in Java, JavaScript, PHP and Python with example libraries.
+There is a link to a list of more libraries for these and other languages below.
+
+.. tabs::
+
+    .. group-tab:: Python
+
+        Using the `pyjwt`_ for python.
+
+        .. code-block:: python
+
+            import jwt
+            import requests
+            import time
+
+            CLIENT_ID = "example"
+            SECRET = "secret"
+
+            payload = {
+                "iss": CLIENT_ID,
+                "iat": int(time.time()),  # current time in seconds
+                "client_id": CLIENT_ID,
+                "user_id": "eample@example.com",
+                "user_representation": "Example Name",
+            }
+            jwt_token = jwt.encode(payload, SECRET, algorithm="HS256")
+
+            # add token token to the authentication HTTP header of your request library
+            zaaktype_url = "https://test.openzaak.nl/catalogi/api/v1/zaaktypen/4acb5ab8-f189-4559-b18a-8a54553a74ff"
```

Comment:
`test.openzaak.nl` is a domain of Maykin test deployment for internal purporses, please change it to something like: `openzaak.gemeente.local` or `openzaak.example.org`

Reply:
Done